homeassistant/components/notion/__init__.py

```diff
@@ -75,6 +75,7 @@ async def async_update() -> dict[str, dict[str, Any]]:
                     f"There was a Notion error while updating {attr}: {result}"
                 ) from result
             if isinstance(result, Exception):
+                LOGGER.debug("".join(traceback.format_tb(result.__traceback__)))
```

Comment:
Can you make this lazy so users who don't have debug logging turned on don't have to pay for it?

It likely doesn't matter so much though since it will only happen on exception

Reply:
You bet!